### ISSUE TITLE
Pages failing WCAG2 AA accessibility testing.

### ISSUE THREAD

**Issue by liujason** [google/web-starter-kit#829]
aXe (a chrome plugin) is reporting 28 violations of WCAG2 AA. 
Most of them are "Elements must have sufficient color contrast", and one instance of "`<ul> must only directly contain <li>, <script> or <template> elements`", in the "Features" tab.

**Comment by addyosmani** [comment]
Is this specifically with the current default template? If so, that would be a Material Design Lite issue. Can you file a bug against http://github.com/google/material-design-lite if this is still an issue?
